common/api/core-common.api.md

```diff
@@ -5762,6 +5762,13 @@ export class NonUniformColor {
     readonly isOpaque: boolean;
 }
 
+// @public

```

Comment:
Let's keep the new APIs `@beta` until we implement normal maps for tiles.

Reply:
Done.